sources/cfsignal/Cargo.toml

```
@@ -0,0 +1,26 @@
+[package]
+name = "cfsignal"
+version = "0.0.1"
+license = "Apache-2.0 OR MIT"
+edition = "2018"
+publish = false
+# Don't rebuild crate just because of changes to README.
+exclude = ["README.md"]
+
+[dependencies]
+log = "0.4"
+serde = { version = "1.0.100", features = ["derive"] }
+simplelog = "0.10"
+snafu = { version = "0.6" }
+toml = "0.5.1"
+tokio = { version = "1", features = ["full"] }
```

This should be pinned to 1.8, the current LTS release for `tokio`, and should turn off any features that aren't needed:
```suggestion
tokio = { version = "~1.8", default-features = false, features = ["macros", "rt-multi-thread"] }  # LTS
```